Test/spec/simd/simd_bitwise.wast

```diff
@@ -0,0 +1,965 @@
+;; Test all the bitwise operators on major boundary values and all special values.
+
+(module
+  (func (export "not") (param $0 v128) (result v128) (v128.not (local.get $0)))
+  (func (export "and") (param $0 v128) (param $1 v128) (result v128) (v128.and (local.get $0) (local.get $1)))
+  (func (export "or") (param $0 v128) (param $1 v128) (result v128) (v128.or (local.get $0) (local.get $1)))
+  (func (export "xor") (param $0 v128) (param $1 v128) (result v128) (v128.xor (local.get $0) (local.get $1)))
+  (func (export "bitselect") (param $0 v128) (param $1 v128) (param $2 v128) (result v128)
+    (v128.bitselect (local.get $0) (local.get $1) (local.get $2))
+  )
+)
+
+;; bitwise operations
+
+;; i8x16
```

Comment:
I think my comment [here](https://github.com/WAVM/WAVM/pull/186#discussion_r321981103) also applies: it doesn't really add much to test these instructions on different forms of the `v128.const` constructor.

Reply:
OK, I will remove these tests.

Reply:
Done.